db/schema.rb

```diff
@@ -678,6 +690,7 @@
   add_foreign_key "blood_sugars", "facilities"
   add_foreign_key "blood_sugars", "users"
   add_foreign_key "communications", "notifications"
+  add_foreign_key "drug_stocks", "facilities"
```

Comment:
Hello there, what's this?

Reply:
I think I've seen this FK come in and out of diffs over the past few weeks as well...not sure if the FK really belongs in our schema or not.

Reply:
Fixed by #2960

Reply:
Nice.